decode/radio_frame.js

```diff
@@ -1,45 +1,43 @@
 var EthernetAddr = require('./ethernet_addr');
 var LogicalLinkControl = require('./llc_packet');
-
+var RadioBeaconFrame = require('./radio_beacon_frame');
 function RadioFrame() {
```

Comment:
There is a small but meaningful perf win from initializing the expected shape of the object in the constructor:

https://gist.github.com/mranney/d383ed6488985ee6185a